### PR TITLE
Regulary cleanup test env

### PR DESCRIPTION
**Pull request by resmo**

### CODE REVIEW

File: .github/workflows/cleanup.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.8'
+
+    - name: Run cleanup
+      run: |
+        pip install cloudscale-cli
+        for resource in objects-user server floating-ip server-group;
```
Comment (href) [inline .github/workflows/cleanup.yml]: I think you might want to add `network` here as well. The current account used in the back has some stray networks left. Though that's generally inconsequential, I would probably add that anyway.

**Review by href — CHANGES_REQUESTED** [review]
See comment about the networks, once that is fixed feel free to merge.

File: .github/workflows/cleanup.yml (hunk 1 above)
Comment (resmo) [reply .github/workflows/cleanup.yml]: good point

**Review by href — APPROVED** [review]
LGTM, merge at will.

**Pull request merged**